spec/services/chat_generator_service_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe ChatGeneratorService do
+  let!(:user) { create(:user) }
+  let(:topic) { create(:topic) }
+  let!(:target) do
+    create(:target, user_id: user.id, topic: topic,
+                    latitude: 34.902425, longitude: 56.177360, radius: 500)
+  end
+
+  subject { ChatGeneratorService.new(user, matches).create }
+
+  context 'when not exists other chat between user and target matches users' do
+    let!(:three_matching_targets) do
+      create_list(:target, 3, topic: topic, latitude: 34.905311, longitude: 56.185386, radius: 900)
+    end
+    let(:matches) { TargetMatchingService.new(target).matches }
+
+    it '.creates a chat per target match' do
```

Comment:
Whats the meaning of the dot ?

Reply:
Nothing, I get confused, and I thought when I was testing create I should write in that way.